### DISCUSSION TITLE
New Modern Themes like in shopify

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#105] [2. Ideas]
I am writing to inquire about the possibility of introducing new, modern designs to the Breeze Theme.

As you may be aware, the demand for modern and trendy designs is constantly increasing in the e-commerce industry. Popular e-commerce platforms like Shopify have already started offering more modern designs, which have proven to be very popular with customers. We believe that introducing new, modern designs to the Breeze Theme would not only keep up with current design trends but also attract more users to the theme.I have added links which most popular themes in shopify

We believe that by creating more modern designs, the Breeze Theme could become even more popular and attract a larger community of users. With more users, we believe that the community could grow even faster, creating a more vibrant ecosystem for Breeze Theme users.

We would greatly appreciate any information or updates you can provide regarding this matter. We understand that creating new designs may take time and resources, but we believe that the potential benefits of doing so are significant.

Thank you for your attention to this request, and we look forward to hearing back from you soon.

https://themes.shopify.com/themes/prestige/styles/allure?price%5B%5D=paid&surface_inter_position=1&surface_intra_position=3&surface_type=all

https://themes.shopify.com/themes/impulse/styles/modern?price%5B%5D=paid&surface_inter_position=1&surface_intra_position=1&surface_type=all

https://themes.shopify.com/themes/symmetry/styles/chantilly?price%5B%5D=paid&surface_inter_position=1&surface_intra_position=4&surface_type=all

Some most popular fashion websites build by Shopify
https://jaxxon.com/
https://www.mvmt.com/
https://vincerocollective.com/

**Comment by dimdimich** [comment]
Hi,

Thank you for your ideas. We are already considering adding new design to Argento theme that will follow similliar styles.  Unfortunatly we can't share any deadlines considering we are still in design phase.
